recipe/meta.yaml

```diff
@@ -41,8 +42,15 @@ test:
 about:
   home: https://github.com/sdv-dev/RDT
   summary: Reversible Data Transforms
+  description: |
+    RDT (Reversible Data Transforms) is a Python library that transforms raw data
+    into fully numerical data, ready for data science. The transforms are reversible,
+    allowing you to convert from numerical data back into your original format.
   license: BUSL-1.1
```

Comment:
The license isn't fully Open Source https://spdx.org/licenses/BUSL-1.1.html and has restrictions for non-production usage. Is it suitable for us?